.github/workflows/release.yml

```diff
@@ -43,14 +45,16 @@ jobs:
       - name: Package release
         id: package_release
         run: |
-          VERSION=`cat semver.txt`
+          dotnet tool restore
+          VERSION=`nbgv get-version --variable Version`
+          NUGETVERSION=`nbgv get-version --variable NuGetPackageVersion`
           OUTPUT=./nupkgs
           echo "##[set-output name=version;]$VERSION"
-          echo "##[set-output name=package_name;]CycloneDX.$VERSION.nupkg"
-          echo "##[set-output name=package_filename;]$OUTPUT/CycloneDX.$VERSION.nupkg"
+          echo "##[set-output name=package_name;]CycloneDX.$NUGETVERSION.nupkg"
+          echo "##[set-output name=package_filename;]$OUTPUT/CycloneDX.$NUGETVERSION.nupkg"
           REPO=cyclonedx/cyclonedx-dotnet
-          dotnet build --configuration Release /p:Version=$VERSION
-          dotnet pack CycloneDX/CycloneDX.csproj --configuration Release /p:Version=$VERSION --output $OUTPUT
+          dotnet build --configuration Release
+          dotnet pack CycloneDX/CycloneDX.csproj --configuration Release --output $OUTPUT
           docker build -f Dockerfile --build-arg VERSION=$VERSION -t $REPO:$VERSION -t $REPO:latest .
```

Comment:
now here i tried to read the file with a 4 segment version in the filename:

```
 [3/3] RUN dotnet tool install --global CycloneDX --version 3.0.6.2 --add-source /tmp/nupkgs &&     ln -s /root/.dotnet/tools/dotnet-CycloneDX /usr/bin/CycloneDX
```
resulting in:
```
#7 1.127 Unhandled exception: Microsoft.DotNet.Cli.NuGetPackageDownloader.NuGetPackageNotFoundException: cyclonedx::3.0.6.2 is not found in NuGet feeds https://api.nuget.org/v3/index.json;/tmp/nupkgs".
#7 1.135    at Microsoft.DotNet.Cli.NuGetPackageDownloader.NuGetPackageDownloader.GetPackageMetadataAsync(String packageIdentifier, NuGetVersion packageVersion, IEnumerable`1 sources, CancellationToken cancellationToken, Boolean includeUnlisted)
#7 1.135    at Microsoft.DotNet.Cli.NuGetPackageDownloader.NuGetPackageDownloader.GetPackageSourceAndVersion(PackageId packageId, NuGetVersion packageVersion, PackageSourceLocation packageSourceLocation, Boolean includePreview, Boolean includeUnlisted, PackageSourceMapping packageSourceMapping)
#7 1.135    at Microsoft.DotNet.Cli.NuGetPackageDownloader.NuGetPackageDownloader.DownloadPackageAsync(PackageId packageId, NuGetVersion packageVersion, PackageSourceLocation packageSourceLocation, Boolean includePreview, Boolean includeUnlisted, Nullable`1 downloadFolder, PackageSourceMapping packageSourceMapping)
#7 1.135    at Microsoft.DotNet.Cli.ToolPackage.ToolPackageDownloader.DownloadAndExtractPackage(PackageLocation packageLocation, PackageId packageId, INuGetPackageDownloader nugetPackageDownloader, String packagesRootPath, IToolPackageStore toolPackageStore, NuGetVersion packageVersion, PackageSourceLocation packageSourceLocation, Boolean includeUnlisted)
#7 1.135    at Microsoft.DotNet.Cli.ToolPackage.ToolPackageDownloader.<>c__DisplayClass8_0.<InstallPackage>b__0()
#7 1.135    at Microsoft.DotNet.Cli.TransactionalAction.Run[T](Func`1 action, Action commit, Action rollback)
#7 1.135    at Microsoft.DotNet.Tools.Tool.Install.ToolInstallGlobalOrToolPathCommand.Execute()
#7 1.135    at System.CommandLine.Invocation.InvocationPipeline.Invoke(ParseResult parseResult)
#7 1.135    at Microsoft.DotNet.Cli.Program.ProcessArgs(String[] args, TimeSpan startupTime, ITelemetry telemetryClient)
#7 ERROR: process "/bin/sh -c dotnet tool install --global CycloneDX --version ${VERSION} --add-source /tmp/nupkgs &&     ln -s /root/.dotnet/tools/dotnet-CycloneDX /usr/bin/CycloneDX" did not complete successfully: exit code: 1
```

Reply:
The nuget packages has version “3.0.6” (package name: CycloneDX.3.0.6.nupkg).